client/extensions/wp-super-cache/cache-stats.jsx

```diff
@@ -55,18 +56,18 @@ class CacheStats extends Component {
 						</tr>
 					</thead>
 					<tbody>
-						{ files.map( ( file, index ) =>
-						<tr className="wp-super-cache__stat" key={ index }>
-							<td className="wp-super-cache__stat-dir">{ file.dir }</td>
-							<td>{ file.files }</td>
-							<td className="wp-super-cache__stat-age">{ getAge( file ) }</td>
-							<td className="wp-super-cache__stat-action">
-								<Button compact>
-									{ translate( 'Delete' ) }
-								</Button>
-							</td>
-						</tr>
-					) }
+						{ map( files, ( file, dir ) => (
+							<tr className="wp-super-cache__stat" key={ file.lower_age }>
```

Comment:
D'oh, this is a leftover from some weird debugging that I did 😳  Good spot, this should be `file.url`.